data/korath/korath missions.txt

```diff
@@ -257,3 +257,81 @@ mission "Small Scale Delivery"
+
+mission "Efret: Quarg Rescue"
+	boarding
+	repeat
+	invisible
+	deadline 1
+	source
+		government "Kor Efret"
+	on offer
+		"old quarg rep" = "reputation: Quarg"
+		# bring it low enough that capping this will result in war
+		"reputation: Quarg" <?= 1
+		conversation
+			branch "unfriendly quarg" # if you're already at war with them
+				"reputation: Quarg" < 0
+			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself from hence, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yonder vessel."`
+			choice
+				`  "I had not realized they were under your protection. I will leave at once."`
+				`  (Ignore the warning.)`
+					goto ignore
+				`  "I think you should leave. This is my prize and I'll take what I please."`
+					goto refuse
+			`  "Leave then anon, ere we change our mind."`
+				accept
+			label ignore
+			action
+				"reputation: Quarg" <?= -1
+			`  "So be it. You shall pay for your transgression against the Efret!" The Quarg begin powering up their weapons...`
+				accept
+			label refuse
+			action
+				"reputation: Quarg" <?= -1
+			`  "What impetuous evil is this! Nay, we shall not allow you your plunder; such may not be tolerated." The Quarg open fire before you have a chance to disengage.`
+				launch
+			label "unfriendly quarg"
+			`Three hostile Wardragons arrive in the system, in response to your act of aggression. One hails you. "Your evil deeds against the Efret shall not be tolerated any longer!"`
+				accept
+	npc evade
+		government "Quarg"
+		to spawn
+			"reputation: Quarg" >= 0
+		personality disables staying entering marked
+		fleet
+			names "quarg"
+			variant
+				"Quarg Wardragon" 2
+	npc evade
+		government "Quarg"
+		to spawn
+			"reputation: Quarg" < 0
+		# they don't just disable.
+		personality staying entering heroic marked nemesis
+		fleet
+			names "quarg"
+			variant
+				"Quarg Wardragon" 3
+	# next time we enter a system
+	on enter
+		conversation
+			branch "already was hostile"
+				"old quarg rep" < 0
+			branch "did not listen to warning"
+				"reputation: Quarg" < 0
+				"old quarg rep" >= 0
+			action
+				"reputation: Quarg" = "old quarg rep"
+			"The Quarg and they seem to be willing to forgive this transgression, this time."
+				decline
+			label "did not listen to warning"
+			action
+				"reputation: Kor Efret" <?= -1
+			"You disregarded the warning of the Quarg... They will probably never forgive you."
+				decline
+			label "already was hostile"
+			"Is seems like the Quarg will not pursue you... But you may want to refrain from assaulting the Efret in the future."
+			action
+				"reputation: Kor Efret" <?= -1
+	destination "Kuwaru Efreti"
```

Comment:
This should be near the top of the mission definition, not hte last thing in it.

Reply:
since it doesnt matter its usually at the end in boarding missions I think?

Reply:
Those ones are always declined, but it is possible to accept this one, and it is possible to land on the destination and complete the mission.

Reply:
I had set it to Earth bruh

Reply:
Was it supposed to be set to Earth?  The comment said that the destination didn't matter

Reply:
Ohh, I see.  Your edit didn't post before I responded.  Yeah, either one works, though the latter seems better to me.

Reply:
Yeah just use to complete never, the destination doesnt matter only if it isnt in Efret space